boards/nordic/nrf52840dk/src/main.rs

```diff
@@ -138,7 +138,7 @@ static mut CHIP: Option<&'static nrf52840::chip::NRF52<Nrf52840DefaultPeripheral
 /// Dummy buffer that causes the linker to reserve enough space for the stack.
 #[no_mangle]
 #[link_section = ".stack_buffer"]
-pub static mut STACK_MEMORY: [u8; 0x1000] = [0; 0x1000];
+pub static mut STACK_MEMORY: [u8; 0x2000] = [0; 0x2000];
```

Comment:
That's a huge stack: 8kB. What needs it?

Reply:
I got a stack overflow when running the UDP example receive application with a 4kB stack -- I think somewhere within the networking code an MTU-sized buffer is being passed on the stack, as I remember running into this issue on Imix as well. I just haven't had time to look into it.

Imix already uses an 8kB stack today, so I changed this to match.